### DISCUSSION TITLE
FCRT school

### DISCUSSION THREAD

**Discussion by Klesone** [hermonochy/Rescueland#30] [Q&A] [answered]
What do FCRT children learn at school?

**Comment by hermonochy** [comment, marked as answer]
FCRT children in the education system receive an education that encompasses a wide range of subjects. Beyond the core academic disciplines that are comparable to those found in Earth’s education system, including mathematics, science, and literature, there is a notable focus on practical fields such as engineering and programming. Recognizing their accelerated learning capabilities, the foundational educational program is designed to last only two years. Following this initial phase, students embark on more specialized training, allowing them to delve deeper into their areas of interest and expertise.
